src/main/resources/static/js/custom.js

```diff
@@ -0,0 +1,20 @@
+$(document).ready(function(){
+    $('.collapsible').collapsible({
+        accordion : false // A setting that changes the collapsible behavior to expandable instead of the default accordion style
+    });
+
+    $('select').material_select();
+
+    $("body").on("submit","#searchForm",function (e) {
+        e.preventDefault();
+
+        $.ajax({
+            type: "POST",
+            url: location.href + "searchResult",
+            data: $(this).serialize(),
+            success: function (data) {
+                $('#Ads').html(data);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["$" is not defined.](https://www.codacy.com/app/matikz1110/PetFinder/file/2644230420/issues/source?bid=3239464&fileBranchId=3321823#l16)